GliaWidgets/CoreSDKClient/CoreSDKClient.Interface.swift

```diff
@@ -89,6 +89,9 @@ struct CoreSdkClient {
         () -> SalemoveSDK.Engagement?
     )
     var getCurrentEngagement: GetCurrentEngagement
+
+    typealias FetchSiteConfigurations = (_ completion: @escaping (Result<SalemoveSDK.Site, Error>) -> Void) -> Void
```

Comment:
Lets use `Result<Self.Site>`, instead of `Result<SalemoveSDK.Site>` - you will need to introduce type alias in CoreSdkClient namespace. Also let's do the same for `SalemoveSDK.Engagement?` on line 89 along with this.